scrapinghub/hubstorage/resourcetype.py

```diff
@@ -54,7 +54,7 @@ def _iter_lines(self, _path, **kwargs):
         kwargs['url'] = urlpathjoin(self.url, _path)
         kwargs.setdefault('auth', self.auth)
         if 'jl' in kwargs:
-            kwargs['data'] = jlencode(kwargs.pop('jl'))
```

Comment:
I think we need a comment here explaining why the `.encode()` is needed and what upstream bug it is preventing.

Reply:
@dangra Good point, updated :ok_hand: